### ISSUE TITLE
initializeVkFFT only once?

### ISSUE THREAD

**Issue by geekac** [DTolm/VkFFT#127]
Hi, DTolm. Thank you for this great work.
I'm using VkFFT for iOS development, so it's metal backend. My problem is that the same size data to perform FFT costs so much time on initializeVkFFT!
so, I want to just call initializeVkFFT once (keep `VkFFTConfiguration configuration` && `VkFFTApplication app`), and send different data (but same size) to `app`.
What should I do? Thank you.

**Comment by DTolm** [comment]
Hello,

Thanks for your interest in VkFFT! Of course, for the same size of system you don't need to initialize the application again. There is a VkFFTLaunchParams struct that is consumed by VkFFTAppend call which can be used to specify different data and offsets.

Best regards,
Dmitrii

**Comment by geekac** [comment]
Thank you, Dmitrii. 
Base on your guidance, I wrote the right code for the same size of data.
But, to learn more, does that means for diffierent size of inputs, I have to  initialize the application again and again?
Best wishes

**Comment by DTolm** [comment]
Yes, different sizes require different codepaths, mainly based on the prime decomposition of the number. There is an option to save the binaries after compilation that works in all backends but Metal, as Metal seems to have its own caching mechanism implemented in it, so secondary initializations should be faster.
